docs/conf.py

```diff
@@ -240,7 +239,7 @@
 intersphinx_mapping = {
     "py3": ("https://docs.python.org/3", None),
     "zopeinterface": ("https://zopeinterface.readthedocs.io/en/latest", None),
-    "api": (apilinks_base_url, "../apidocs/objects.inv"),
+    "api": (api_base_url, "../apidocs/objects.inv"),
```

Comment:
Maybe rename "api" with "twisted" ... to know that this is the twisted on API...
I am not sure... "api" looks very generic... but at the same time, this is just a random name.

Reply:
The names where can be used in references for disambiguation, but since the API objects have unique names anyway, this isn't used.

Since this inventory only contains API objects and not narrative objects, "api" is specific about which part of Twisted it covers, although the fact that it covers Twisted in the first place is implicit.